Resources/Prototypes/_Goobstation/Entities/Clothing/Back/duffel.yml

```
@@ -0,0 +1,8 @@
+﻿- type: entity
+  parent: [ClothingBackpackDuffel, BaseCentcommContraband]
+  id: ClothingBackpackDuffelBlueshield
+  name: blueshield duffel bag
+  description: A very protective duffel bag, smells like energy gun.
+  components:
+  - type: Sprite
```

yamllint (1.35.1)

[error] wrong indentation: expected at least 3

(indentation)